iFixit/core-primitives
# core-primitives: adds css variables, font stacks, font options

Summary of changes
---
**Adds:**
- css variable version of primitives
- font stacks
- font options
- modifies specific line-height values to match existing

QA
---
I threw together this test to help confirm this is working as expected: https://codesandbox.io/s/css-variables-test-o9ppw

It links to a static built version of core-components.css I uploaded, located here: https://files-889os7k37.vercel.app/core-primitives.css

Merge checklist
---

- [x] Bump the package version by running `npm version [patch | minor | major]` from the command line (if applicable).

> **Note:** In the context of Core Primitives, significant changes to the library or workflow, or removing primitives would be considered a major update, adding or updating primitives would be considered a minor update, and fixing primitives would be considered a patch. Non-code changes (e.g. documentation) do not require a version bump.
